content/intro-to-storybook/svelte/en/screen.md

```diff
@@ -0,0 +1,161 @@
+
+In this chapter we continue to increase the sophistication by combining components in a screen and developing that screen in Storybook.
+
+## Nested container components
+
+As our app is very simple, the screen we’ll build is pretty trivial, simply wrapping the `TaskList` component (which supplies its own data via Svelte Store) in some layout and pulling a top-level `error` field out of the store (let's assume we'll set that field if we have some problem connecting to our server). Create `InboxScreen.svelte` in your `components` folder:
+
+```html
+<!-- src/components/InboxScreen.svelte-->
+
+<script>
+  import TaskList from "./TaskList.svelte";
+  export let error=false;
+</script>
+
+<div>
+  {#if error}
+    <div class="page lists-show">
+      <div class="wrapper-message">
+        <span class="icon-face-sad" />
+        <div class="title-message">Oh no!</div>
+        <div class="subtitle-message">Something went wrong</div>
+      </div>
+    </div>
+  {:else}
+    <div class="page lists-show">
+      <nav>
+        <h1 class="title-page">
+          <span class="title-wrapper">Taskbox</span>
+        </h1>
+      </nav>
+      <TaskList/>
+    </div>
+  {/if}
+</div>
+```
+
+We need to update our store (in `src/store.js`) to include our new `error` field, transforming it into :
+
+```javascript
+import { writable } from "svelte/store";
+const TaskBox = () => {
+  const { subscribe, update } = writable([
+    { id: "1", title: "Something", state: "TASK_INBOX" },
+    { id: "2", title: "Something more", state: "TASK_INBOX" },
+    { id: "3", title: "Something else", state: "TASK_INBOX" },
+    { id: "4", title: "Something again", state: "TASK_INBOX" }
+  ]);
+
+  return {
+    subscribe,
+    archiveTask: id =>
+      update(tasks => {
+        tasks.map(task =>
+          task.id === id ? { ...task, state: "TASK_ARCHIVED" } : task
+        );
+        return tasks;
+      }),
+    pinTask: id =>
+      update(tasks => {
+        tasks.map(task =>
+          task.id === id ? { ...task, state: "TASK_PINNED" } : task
+        );
+        return tasks;
+      })
+  };
+};
+export const taskStore = TaskBox();
+
+// store to handle the app state
+const appState = () => {
+  const { subscribe, update } = writable(false);
+  return {
+    subscribe,
+    error: () => update(error => !error)
+  };
+};
+
+export const AppStore = appState();
+```
+We also change the `App` component to render the `InboxScreen` (eventually we would use a router to choose the correct screen, but let's not worry about that here):
+
+```html
+<!-- src/App.svelte-->
+<script>
+  import { AppStore } from "./store";
+  import InboxScreen from "./components/InboxScreen.svelte";
+</script>
+
+<InboxScreen error={$AppStore} />
```

Comment:
After doing this I found the app rendered correctly but I couldn't actually change the state of tasks.

Reply:
https://github.com/tmeasday/learnstorybook-svelte/commit/491504148e1def613c8ffc55ea4215fcf30a3194

Reply:
The data and screen code needs a bit of a fix. I've tracked down the error and i'll update the code tomorrow and commit it if you don't mind.

Reply:
The store code was updated to correctly display the data and allow the state changes with the latest commit